apps/design/frontend/src/export_screen.tsx

```diff
@@ -176,22 +211,43 @@ export function ExportScreen(): JSX.Element | null {
         <H2>Ballots</H2>
         <Column style={{ gap: '1rem' }}>
           {features.CHOOSE_BALLOT_TEMPLATE && (
-            <InputGroup label="Ballot Template">
-              <SearchSelect
-                aria-label="Ballot Template"
-                value={ballotTemplateId}
-                options={Object.entries(ballotTemplateOptions).map(
-                  ([value, label]) => ({ value, label })
-                )}
-                onChange={(value) => {
-                  setBallotTemplateMutation.mutate({
-                    electionId,
-                    ballotTemplateId: value as BallotTemplateId,
-                  });
-                }}
-                disabled={Boolean(ballotsFinalizedAt)}
-              />
-            </InputGroup>
+            <React.Fragment>
+              <InputGroup label="Ballot Template">
+                <SearchSelect
+                  aria-label="Ballot Template"
+                  value={ballotTemplateId}
+                  options={Object.entries(ballotTemplateOptions).map(
+                    ([value, label]) => ({ value, label })
+                  )}
+                  onChange={(value) => {
+                    setBallotTemplateMutation.mutate({
+                      electionId,
+                      ballotTemplateId: value as BallotTemplateId,
+                    });
+                  }}
+                  disabled={Boolean(ballotsFinalizedAt)}
+                />
+              </InputGroup>
+              {ballotTemplateId === 'NhBallot' && !electionInfo.signature && (
+                <Row
+                  style={{
+                    alignItems: 'center',
+                    height: '100%',
+                  }}
+                >
+                  <Callout color="danger" icon="Danger">
+                    <span>
+                      The election is missing the following required field:
+                      &quot;signature&quot;. Update in{' '}
+                      <Link to={routes.election(electionId).electionInfo.path}>
+                        Election Info
+                      </Link>
+                      .
+                    </span>
+                  </Callout>
+                </Row>
+              )}
+            </React.Fragment>
```

Comment:
I think switching from VX to NH template for an existing election is going to be a rare enough edge case that this warning isn't necessary (given that it's already guarded against in the places where we render the ballot template)